CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [5.3.3] - 2020-08-18
+### Changed
+- Changed publish to RubyGems requirements to avoid faulty publications
+- Changed release process in `CONTRIBUTING.md`
```

Comment:
Please see our [changelog guidelines](https://github.com/cyberark/community/blob/master/Conjur/CONTRIBUTING.md#changelog-guidelines) - I don't think this addresses the user-impact of this change and it does not include a link to a github issue.

Suggested message that addresses user-impact would be something like:
- Release process is updated to ensure that the published Ruby Gem matches a tag in this repository, so that consumers of this gem can always reference the correct source code included in any given version.